buildSrc/src/main/java/com/uber/okbuck/core/manager/RobolectricManager.java

```diff
@@ -82,7 +82,7 @@ public void finalizeDependencies() {
     API_25("7.1.0_r7", "r1"),
     API_26("8.0.0_r4", "r1"),
     API_27("8.1.0", "4611349"),
-    API_28("P", "4651975");
+    API_28("9", "4799589");
```

Comment:
We cannot do this for backwards compatibility reasons. Lets create a new enum API_28 and move the existing one to API_P so we can get both the jars

Reply:
Good point @kageiit, just fixed it.